Cargo.toml

```diff
@@ -49,10 +49,12 @@ path = "benches/macrobenches.rs"
 [features]
 default = []
 
-## Enables support for asynchronous reading from `tokio`'s IO-Traits by enabling
+## Enables support for asynchronous reading and writing from `tokio`'s IO-Traits by enabling
 ## [reading events] from types implementing [`tokio::io::AsyncBufRead`].
 ##
 ## [reading events]: crate::reader::Reader::read_event_into_async
+##
+## [writing events] to buffers implementing [`tokio::io::AsyncWrite`].
```

Comment:
The `##` doc comments is usual rustdoc markdown comments, which is processed by `document-features` crate. So please rewrite that according to that. If you want to make hyperlink on `writing events`, please add hyperlink target

Reply:
I think I maybe misunderstood the docs there so I removed that change.